src/System.IO.FileSystem/src/project.json

```diff
@@ -23,14 +23,14 @@
         "dotnet5.4"
```

Comment:
No reason, I removed one by accident I think and mixed them up. Which imports? Didn't think it mattered.